### ISSUE TITLE
Projections Not Working In Mongo Scrapbooks

### ISSUE THREAD

**Issue by burkeholland** [Microsoft/vscode-cosmosdb#214]
It looks like projections are being ignored in Mongo Scrapbooks. Instead, the entire document is returned. To replicate...

### Reproduce

Insert the following record....

```
db.heroes.insert({
  "id": 2,
  "name": "Batman",
  "saying": "I'm Batman"
})
```

Create a new Mongo DB Scrapbook and run the following query...

```
db.heroes.find({ id: 2 }, { saying: 1 })
```

### Expected Result

```
{ "_id" : "59f8f9179c16bb075e1da18e", "saying" : "I'm Batman" }
```

### Actual Result

```
{
  "_id": "59f8f9269c16bb075e1da18f",
  "id": 2,
  "name": "Batman",
  "saying": "I'm Batman",
  "__v": 0
}
```

**Note that this query returns the proper result when run from the Mongo Shell**

**Comment by StephenWeatherford** [comment]
Moved to 0.6.0.

**Comment by PrashanthCorp** [comment]
Marking as closed, fixed by #460
